### PR TITLE
Feature/historical fires update - dashboards + map + topics

### PR DESCRIPTION
**Pull request by mbarrenechea**
## Overview

This PR adds several new widgets to the fires tab, aw well as a layer for VIIRS fires:

- Ranked Admin widget
- Seasonal Fires widget chart (w/ statistical bands)
- Cumulative Fires widget chart (w/ statistical bands)
- Fires within/without widget pie

### Ongoing to do

- [x] Add 'Select indicator' button to pie (a la 'Select comparison year' button)
- [x] Implement date selector in legend
- [x] QA Pie and Rank widget show on map buttons?

### Done

- [x] Move widgets out of forest change to fire tab
- [x] Widget structure
- [x] Widget filters
  - [x] Confidence level
  - [x] Forest type: 
    - [x] Intact forest landscapes
    - [x] Plantations
    - [x] Primary forests
  - [x] Land category: 
    - [x] Alliance for Zero extintion
    - [x] Indigenous lands
    - [x] Key biodiversity areas
    - [x] Mining concessions
    - [x] Protected areas
- [x] Widget fetch from correct datasets @01painadam `app/javascript/data/analysis-datasets.json`
- [x] Implement dataset selector in settings (MODIS/VIIRS)
- [x] Variable std dev
- [x] Update sentence
- [x] Layer
- [x] Set minimum brush width to 4 weeks
- [x] Placeholder for the comparison year
- [x] Labels
- [x] Add comparison year prompt
- [x] Hover on dataset info button
- [x] Add Fires Tab.

Screen Shot 2020-04-27 at 10 06 40

## Testing

- Go to the dashboard page and select one country

### CODE REVIEW

File: app/javascript/services/analysis-cached.js [hunk 1]
```diff
@@ -699,7 +742,7 @@ export const fetchGladAlerts = ({
     adm1,
     adm2,
     grouped,
-    glad: true
+    allowedParams: 'glad'
```
Comment (edbrett) [inline app/javascript/services/analysis-cached.js:745]: Better. Thank you.

File: app/javascript/services/analysis-cached.js [hunk 2]
```diff
@@ -775,6 +818,101 @@ export const fetchGLADLatest = () => {
...
+        alerts: d.alert__count,
+        area_ha: d.alert_area__ha
+      }))
+    }
+  }));
+};
+
+export const fetchVIIRSLatest = () => {
+  const url =
+    'https://d20lgxzbmjgu8w.cloudfront.net/nasa_viirs_fire_alerts/latest/max_alert__date';
```
Comment (edbrett) [inline app/javascript/services/analysis-cached.js:891]: Interesting url.

File: app/javascript/services/analysis-cached.js (hunk 1 above)
Comment (01painadam) [reply app/javascript/services/analysis-cached.js:745]: > You seem to have got it all working! Nothing wrong with any of the analysis building. I have the dynamic sentences not working on staging but maybe this isnt ready yet?

Not yet, no - need some new helper functions that depend on the min/max dates of the brush.

File: app/javascript/services/analysis-cached.js (hunk 2 above)
Comment (01painadam) [reply app/javascript/services/analysis-cached.js:891]: It's a temp url for the moment.

**Comment by edbrett** [comment]
Now with feature flag for staging. Im going to merge into develop to prevent issues with features coming in the June contract.

**Pull request merged**